client/src/pages/attachments/Form.js

```diff
@@ -247,7 +248,12 @@ const AttachmentForm = ({ edit, title, initialValues }) => {
 
   function save(values, form) {
     const attachment = Attachment.filterClientSideFields(values, "content")
+    const updatedRelatedObjects = relatedObjects.map(attach => ({
+      relatedObjectType: attach.relatedObjectType,
+      relatedObjectUuid: attach.relatedObjectUuid
+    }))
     attachment.classification = values.classification
+    attachment.attachmentRelatedObjects = updatedRelatedObjects
```

Comment:
The form parameter is unused; instead I suggest passing the related objects as a parameter. Attachment never contains a "content" field. Also the reassignment of `attachment.classification` can be removed. Code can also be simplified a bit by using destructuring, and removing the unnecessary `const` by assigning it directly to the attachment:
```
  function save(values, attachmentRelatedObjects) {
    const attachment = Attachment.filterClientSideFields(values)
    attachment.attachmentRelatedObjects = attachmentRelatedObjects.map(
      ({ relatedObjectType, relatedObjectUuid }) => ({
        relatedObjectType,
        relatedObjectUuid
      })
    )
```